packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -411,6 +411,16 @@ const vpc = [
       )
     );
   }),
+  rest.get('*/vpcs/:vpcId', (req, res, ctx) => {
+    return res(
+      ctx.json(
+        vpcFactory.build({
+          description: `VPC for webserver and database. VPC for webserver and database. VPC for webserver and database. VPC for webserver and database. VPC for webserver VPC for webserver VPC for webserver VPC for webserver VPC for webserver.VPC for webserver and database. VPC for webserver and database. VPC for webserver and database. VPC for webserver and database. VPC for webserver VPC for webserver VPC for webserver VPC for webserver VPC for webserver.`,
```

Comment:
The API enforces a max character count of 255 characters for the `description` field, we should probably just match that:

```suggestion
          description: `VPC for webserver and database. VPC for webserver and database. VPC for webserver and database. VPC for webserver and database. VPC for webserver VPC for webserver VPC for webserver VPC for webserver VPC for webserver.VPC for webserver and database!!! VPC`,
```